pootle/apps/pootle_store/views.py

```diff
@@ -390,8 +391,7 @@ def get_units(request):
             # Since `extra()` has been used before, it's necessary to
             # explicitly request the `store__pootle_path` field. This is a
             # subtetly in Django's ORM.
-            uid_list = [u['id'] for u
-                        in step_queryset.values('id', 'store__pootle_path')]
+            uid_list = list(step_queryset.values_list('id', flat=True))
```

Comment:
Sure, but since you are removing the `store__pootle_path` field now, it makes sense to adjust the comment now. Otherwise when reviewing in the future we will have to check history to know what is the comment talking about. So I would kindly request this to be replaced with a simple `values_list` in this PR, and also properly adjust or remove these comments.